### ISSUE TITLE
Error joining/creating team

### ISSUE THREAD

**Issue by Morganamilo** [FTBTeam/FTB-Continuum#66]
* **Modpack Version**: v1.0.2

* **Issue**:  When logging into a server, a message appears 

```
You havn't joined or created a team yet! [Click Here] to open TeamsGUI or [Click Here] to hide this message.
```

Clicking the first prompt gives.

```
Invalid subcommand `gui`! Avaliable subcommands: kick, leave, transfer_ownership, create, join, set_status, config and request_invite
```

Using the `/team` command or clicking the team button in the top left in the inventory screen both work.

* **Is it Repeatable?**: Happens after restarting, have not tested with other people.

* **Mod/s Affected**: FTBUtilities

**Comment by tfox83** [comment]
I let the moddev know about this, thanks for reporting.

**Comment by LatvianModder** [comment]
Fixed in 5.1+

**Comment by tfox83** [comment]
Fixed in Dev
